module/observations/helper.js

```diff
@@ -1048,4 +1052,159 @@ module.exports = class ObservationsHelper {
         });
     }
 
+
+         /**
+      * observation link.
+      * @method
+      * @name getObservationLink
+      * @param  {String} observationSolutionId observation solution external Id.
+      * @param  {String} appName name of app.
+      * @returns {getObservationLink} observation getObservationLink.
+     */
+
+    static getObservationLink(observationSolutionId, appName) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let link = null;
+                
+                let observationData = await solutionHelper.solutionDocuments({
+                        externalId : observationSolutionId,
+                        isReusable : false,
+                        type : messageConstants.common.OBSERVATION
+
+                        },[
+                            "link"
+                    ]);
+
+                if(!observationData) {
+                    throw new Error(messageConstants.apiResponses.OBSERVATION_NOT_FOUND);
+                }
+
+                let appDetails = await kendraService.getAppDetails(appName);
+                var jsonAppDetails = JSON.parse(appDetails)
```

Comment:
@priyanka-TL  Parse json in kendraService.getAppDetails helper functionality only.